Alertinator.php

```diff
@@ -35,11 +35,13 @@ class Alertinator {
 
    protected $_twilio;
 
-   public function __construct($config) {
+   public function __construct($config, alertLogger $logger = NULL) {
```

Comment:
we usually use the lower case null but maybe this file has a different style?

Reply:
yeah I'll change that next commit